packages/story-editor/src/components/keyboardShortcutsMenu/index.js

```diff
@@ -55,41 +57,29 @@ function KeyboardShortcutsMenu() {
     })
   );
 
-  useEffect(() => {
-    if (isKeyboardUser() && !isOpen) {
-      // When menu closes, return focus to toggle menu button
-      anchorRef.current.focus?.();
-    }
-  }, [isOpen]);
-
   useGlobalKeyDownEffect(TOGGLE_SHORTCUTS_MENU, toggle, [toggle]);
   useFocusOut(wrapperRef, close, [close]);
 
   return (
     <DirectionAware>
       <Wrapper ref={wrapperRef}>
-        <Tooltip
-          title={__('Toggle Keyboard Shortcuts', 'web-stories')}
-          placement={TOOLTIP_PLACEMENT.TOP}
-          shortcut="mod+/"
-          hasTail
+        <Popup
+          popupId="keyboard_shortcut_menu"
+          isOpen={isOpen}
+          ariaLabel={__('Keyboard Shortcuts', 'web-stories')}
         >
-          <Button
-            ref={anchorRef}
-            variant={BUTTON_VARIANTS.SQUARE}
-            type={BUTTON_TYPES.TERTIARY}
-            size={BUTTON_SIZES.SMALL}
-            aria-label={__('Keyboard Shortcuts', 'web-stories')}
-            aria-haspopup
-            aria-expanded={isOpen}
-            onClick={toggle}
-          >
-            <Icons.Keyboard />
-          </Button>
-        </Tooltip>
-        <Popup isOpen={isOpen}>
           <ShortcutMenu toggleMenu={toggle} />
         </Popup>
+
+        <ToggleButton
+          isOpen={isOpen}
+          aria-owns="keyboard_shortcut_menu"
+          onClick={toggle}
+          aria-label={__('Keyboard Shortcuts', 'web-stories')}
+          label={__('Keyboard Shortcuts', 'web-stories')}
+          MainIcon={MainIcon}
+          shortcut="mod+/"
```

Comment:
🤔 does this need to be wrapped in a `kbd` for accessibility?

Reply:
🤔  great question! so, the focused label when tooltips show up is either the button's text or the aria label. I don't think it's necessary to wrap it in a `kbd` because the user's already interacting with the element and doesn't need the shortcut.